src/components/DataTable.vue

```diff
@@ -97,7 +97,14 @@
         class="px-5 py-2"
         style="background-color: #f5f5f5"
       >
-        <h3 class="my-3">{{ item.Title }}</h3>
+        <h3 class="my-3">
+          {{ item.Title
+            ? item.Title
+            : (item.Description?.length > 100
+                ? (item.Description.slice(0, 100) + '...')
+                : item.Description)
```

Comment:
Since this logic is on two places, I would prefer to create a utility function for the `displayTitle`

Reply:
yeah that makes sense.